src/index.tsx

```diff
@@ -111,13 +114,18 @@ const startApp = async () => {
                             checkout={checkout}
                             apolloClient={apolloClient}
                           >
-                            <Switch>
-                              <Route
-                                path={checkoutBaseUrl}
-                                component={CheckoutApp}
-                              />
-                              <Route component={App} />
-                            </Switch>
+                            <ThemeProvider theme={defaultTheme}>
```

Comment:
ThemeProvider is already used in `src/App`

Reply:
Yes, but now styled components are not working for checkout. I will remove it, but we should think about moving Theme Provider somewhere else.